scaleway/data_source_rdb_instance_test.go

```diff
@@ -1,31 +1,35 @@
 package scaleway
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestAccScalewayDataSourceRdbInstance_Basic(t *testing.T) {
 	tt := NewTestTools(t)
 	defer tt.Cleanup()
+	// instance name should be unique
```

Comment:
Why? In particular, why should it be unique and random?

Reply:
Well, I'm not available to reproduce this error. I'll rollback to not random name